configs/ote/custom-sematic-segmentation/ocr-lite-hrnet-18/pot_optimization_config.json

```diff
@@ -0,0 +1,92 @@
+{
+    "algorithms": [
+        {
```

Comment:
Is cant used json like POT config format without "algorithms"? Or can there be another parameters here?

Reply:
Algorithms in POT are always in a list. Can't start json config with a list.

Reply:
ok